libcodechecker/server/run_db_model.py

```diff
@@ -70,6 +70,25 @@ def mark_finished(self):
         self.duration = ceil((datetime.now() - self.date).total_seconds())
 
 
+class RunHistory(Base):
+    __tablename__ = 'run_histories'
+
+    id = Column(Integer, autoincrement=True, primary_key=True)
+    run_id = Column(Integer,
+                    ForeignKey('runs.id', deferrable=True,
+                               initially="DEFERRED", ondelete='CASCADE'),
+                    index=True)
+    tag = Column(String)
```

Comment:
Tags need not be unique then?

Reply:
If it would be unique, how would I add the same tag for the same or for a different run?
E.g:

| run_id | tag | user | date |
| --------- | ---- | ------ | ------ |
1	| v1.0 | Anonymous | 2017-09-06 16:56:33.681858
1	| v1.0 | Anonymous | 2017-09-06 17:56:33.681858

The question is, do we want to allow the user to do this?

Reply:
Let's ask the others. @dkrupp @gyorb Depends on whether we want to consider this annotation as a _tag_ or as a _comment_. Tags, such as version tags, are usually unique.

Reply:
the tag should be unique at least within a run.

Reply:
Maybe the name using the name `version tag` would be more obvious.